### PR TITLE
fix(board view): Set task as the default work item in board view. Fixes #1687.

### LINKED ISSUE #1687: Board view should default to "Task" WI type
When I am looking at the board view, I am usually working within a sprint and moving my tasks from one column to another.  Therefore, we should default the board view to the work item type "Tasks" for now until we can solve the board view having many types shown.

### PR DESCRIPTION
**Pull request by nimishamukherjee**
fix(board view): Set task as the default work item in board view. Fixes #1687.

### CODE REVIEW

File: src/app/work-item/work-item-board/work-item-board.component.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <section>
       <toolbar-panel
         context="boardview"
-        [wiTypes]="workItemTypes"
+        [wiTypes]="workItemTypesToolbar"
```
Comment (sanbornsen) [inline src/app/work-item/work-item-board/work-item-board.component.html]: You can use the same old one and in ngOnChanges of toolbar-panel component you can do the filtering.

Comment (nimishamukherjee) [inline src/app/work-item/work-item-board/work-item-board.component.html]: How do we set this.currentBoardType in the board component?

**Comment by fabric8cd** [comment]
@nimishamukherjee snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1820-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@nimishamukherjee snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1820-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@nimishamukherjee snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1820-fabric8-planner.int.rdu2c.fabric8.io

File: src/app/work-item/work-item-board/work-item-board.component.html (hunk 1 above)
Comment (sanbornsen) [inline src/app/work-item/work-item-board/work-item-board.component.html]: I think we should focus on process templates for these kind of features.

**Comment by rawlingsj** [comment]
fabric8 CI seems to be failing for the same reason as https://github.com/fabric8io/fabric8-planner/pull/1889

```
fabric8-ui/src/fabric8-planner/src/app/work-item/work-item.service.json doesn't exist
```

**Comment by nimishamukherjee** [comment]
Closing this as it was a hack targeted at Summit.

**Pull request closed without merging**